content/webapp/pages/search/index.tsx

```diff
@@ -161,45 +204,63 @@ const NewSearchPage: NextPageWithLayout<NewProps> = ({
           <SearchNoResults query={queryString} />
         </Container>
       ) : (
-        <Container style={{ display: 'flex' }}>
-          <BasicSection>
-            <Container>
-              <SectionTitle sectionName="Content results" />
-              <p>{contentResults?.totalResults || 0} results</p>
-              <div
-                style={{
-                  fontSize: '12px',
-                  maxWidth: '780px',
-                  overflow: 'scroll',
-                }}
-              >
-                {contentResults && (
-                  <pre>{JSON.stringify(contentResults, null, 2)}</pre>
-                )}
+        <BasicSection>
+          <Container>
+            <p className={font('intr', 5)}>
+              {contentQueryFailed ? (
+                <>
+                  There was a problem fetching some search results. Please try
+                  again. If the problem persists, please contact us.
+                </>
+              ) : (
+                <>
+                  {totalResults} result
+                  {totalResults === 1 ? '' : 's'} for{' '}
+                  <span className={font('intb', 6)}>{queryString}</span>
```

Comment:
If no query is entered this looks incomplete

Reply:
ah good point – will update, thanks